sharktank/tests/models/flux/flux_test.py

```diff
@@ -230,32 +233,26 @@ def testExport(self):
             qkv_bias=True,
             guidance_embed=False,
         )
-        theta = make_random_theta()
+        theta = make_random_theta(dtype)
         theta = self.save_load_theta(theta)
         flux = FluxModelV1(
             theta=theta,
             params=params,
         )
 
-        img = torch.rand([self.batch_size, 1024, 64])
-        img_ids = torch.rand([self.batch_size, 1024, 3])
-        txt = torch.rand([self.batch_size, 512, 4096])
-        txt_ids = torch.rand([self.batch_size, 512, 3])
-        timesteps = torch.rand([self.batch_size])
-        y = torch.rand([self.batch_size, 768])
-
-        flux.forward(img, img_ids, txt, txt_ids, timesteps, y)
-        fxb = aot.FxProgramsBuilder(flux)
-
-        @fxb.export_program(
-            name="flux", args=(img, img_ids, txt, txt_ids, timesteps, y), strict=False
+        export_flux_transformer_model_mlir(
+            flux,
+            output_path=self._temp_dir / "model.mlir",
+            batch_sizes=[self.batch_size],
         )
-        def _(model, img, img_ids, txt, txt_ids, timesteps, y) -> torch.Tensor:
-            return model.forward(img, img_ids, txt, txt_ids, timesteps, y)
 
-        output = aot.export(fxb)
-        output.verify()
-        asm = str(output.mlir_module)
+    @with_flux_data
+    def testExportSchnellFromHuggingFace(self):
+        export_flux_transformer_from_hugging_face(
+            "black-forest-labs/FLUX.1-schnell/black-forest-labs-transformer",
```

Comment:
Ideally we should be using the official black-forest-labs/FLUX.1-dev model. I know it is a gated repo but we should be able to use github secrets with an authentication token

Reply:
There were some license concerns, but I think they got resolved. I can follow up with another PR that adds it.